rtp.h

```diff
@@ -107,6 +107,14 @@ const char *janus_rtp_header_extension_get_from_id(const char *sdp, int id);
  * @returns 0 if found, -1 otherwise */
 int janus_rtp_header_extension_parse_audio_level(char *buf, int len, int id, int *level);
 
+/*! \brief Helper to parse a ssrc-audio-level RTP extension (https://tools.ietf.org/html/rfc6464)
+ * @param[in] buf The packet data
+ * @param[in] len The packet data length in bytes
+ * @param[in] id The extension ID to look for
+ * @param[out] level The level value in dBov (0=max, 127=min)
+ * @returns dBov level of audio, 0 otherwise */
+int janus_rtp_header_extension_parse_audio_level_and_return(char *buf, int len, int id, int *level);
+
```

Comment:
Hi @lminiero, thanks for the feedback, I guess merge done wrong, I'll try to rebase to master.

Strange thing is that I don't receive notifications on replies, i need to enable this.